src/content/deployment/subctl/_index.en.md

```diff
@@ -28,20 +28,21 @@ In addition, it generates a `broker-info.subm` file which can be used with the `
 contains the following details:
 
 * Encryption PSK key
-* Broker access details for subsequent `subctl` runs.
-* Service discovery settings
-* Globalnet settings
+* Broker access details for subsequent `subctl` runs
+* Service Discovery settings
 
 #### deploy-broker flags
 
-| Flag                               | Description
-|:-----------------------------------|:----------------------------------------------------------------------------|
-|`--kubeconfig` `<string>`           | Absolute path(s) to the kubeconfig file(s) (default "$HOME/.kube/config")
-|`--kubecontext` `<string>`          | kubeconfig context to use
-|`--service-discovery`               | Enable Multi Cluster Service Discovery
-|`--globalnet`                       | Enable support for Overlapping cluster/service CIDRs in connecting clusters (default disabled)
-|`--globalnet-cidr-range` `<string>` | Global CIDR supernet range for allocating GlobalCIDRs to each cluster (default "169.254.0.0/16")
-|`--ipsec-psk-from` `<string>`       | Import IPSEC PSK from existing Submariner broker file, like broker-info.subm (default "broker-info.subm)
+<!-- markdownlint-disable line-length -->
+| Flag                                  | Description
+|:--------------------------------------|:---------------------------------------------------------------------------------------------------|
+| `--kubeconfig` `<string>`             | Absolute path(s) to the kubeconfig file(s) (default "$HOME/.kube/config")
+| `--kubecontext` `<string>`            | kubeconfig context to use
+| `--service-discovery`                 | Enable Multi Cluster Service Discovery (default true)
+| `--globalnet`                         | Enable support for overlapping Cluster/Service CIDRs in connecting clusters (default disabled)
+| `--globalnet-cidr-range` `<string>`   | Global CIDR supernet range for allocating GlobalCIDRs to each cluster (default "169.254.0.0/16")
+| `--ipsec-psk-from` `<string>`         | Import IPsec PSK from existing Submariner broker file, like broker-info.subm (default "broker-info.subm)
```

Comment:
Your change didn't introduce it, but I just noticed there's a missing end-quote here.

```suggestion
| `--ipsec-psk-from` `<string>`         | Import IPsec PSK from existing Submariner broker file, like broker-info.subm (default "broker-info.subm")
```

Reply:
Done in the latest, thanks @nyechiel!